README.md

```diff
@@ -1,13 +1,22 @@
 # Food Oasis LA
 This is the repo for the backend of [Food Oasis LA](https://foodoasisla.github.io/).
 
-## Proposed Technology Stack
+## Technology Stack
   - Language: Python 3.5
   - Datastore: Postgres
   - Framework: [Django-Rest](http://www.django-rest-framework.org/)
   - [VirtualEnv](http://docs.python-guide.org/en/latest/dev/virtualenvs/)
   - Deployment: Heroku
 
+## Setup
+Detailed instructions are available in CONTRIBUTING.md
+Quickstart
+1) Install Postgres and start server.
+2) Clone app `git clone https://github.com/foodoasisla/backend.git`
+3) Create and activate virtualenv in /backend.
+4) Install dependencies `pip install -r requirements.txt`
+5) Run Makefile `./Makefile`. (you may need to make the file executable `chmod +x Makefile`)
```

Comment:
ah.....makes sense. OK, that explains a couple of things. I'll be there tonight and we can talk more. I think I fundamentally misunderstood what was going on with the makefile :)